src/scripts/item-review/item-review.html

```diff
@@ -1,3 +1,26 @@
+<div ng-if="$ctrl.canReview == false">
+  <form class="settings">
+    <div class="review-setting-table">
+      <div ng-show="$ctrl.featureFlags.reviewsEnabled">
+        <div class="review-setting-table--name">
+          <label for="showReviews" translate-attr="{ title: 'Settings.ShowReviewsHelp'}" translate="Settings.ShowReviews"></label>
+        </div>
+        <div class="review-setting-table--value">
+          <input type="checkbox" id="showReviews" ng-change="$ctrl.valueChanged()" ng-model="$ctrl.settings.showReviews"/>
+        </div>
+      </div>
+      <div ng-show="$ctrl.featureFlags.reviewsEnabled">
+        <div class="review-setting-table--name">
+          <label for="allowIdPostToDtr" translate-attr="{ title: 'Settings.AllowIdPostToDtrHelp'}" translate="Settings.AllowIdPostToDtr"></label>
```

Comment:
Can you add the whitespace to this? `{ title: 'Settings.AllowIdPostToDtrHelp' }`